build-system/config.js

```diff
@@ -139,6 +139,7 @@ module.exports = {
     // Files in this testdata dir are machine-generated and are not part
     // of the AMP runtime, so shouldn't be checked.
     '!extensions/amp-a4a/*/test/testdata/*.js',
+    '!examples/*.js',
```

Comment:
let's me more specific I see examples folder has `pwa.js` file that is passing presubmit check.